js/binance.js

```diff
@@ -1889,7 +1900,7 @@ module.exports = class binance extends Exchange {
         const uppercaseType = type.toUpperCase ();
         const validOrderTypes = this.safeValue (market['info'], 'orderTypes');
         if (!this.inArray (uppercaseType, validOrderTypes)) {
-            throw new InvalidOrder (this.id + ' ' + type + ' is not a valid order type in ' + market['type'] + ' market ' + symbol);
+            throw new InvalidOrder (this.id + ' ' + type + ' is not a valid order type in market ' + symbol);
```

Comment:
```suggestion
            throw new InvalidOrder (`${this.id} ${type} is not a valid order type in market ${symbol}`);
```

Reply:
@mrbaguvix that would not be transpileable and would most likely break the build due to these details:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#multilanguage-support
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes